test/Conversion/LTLToCore/assertproperty.mlir

```diff
@@ -0,0 +1,42 @@
+// RUN: circt-opt %s --lower-ltl-to-core | FileCheck %s
+// Tests that an Assert Property high level statement can be converted correctly
+
+module {
+  //CHECK:  hw.module @test(in %clock : !seq.clock, in %reset : i1, in %0 "" : i1)
+  hw.module @test(in %clock : !seq.clock, in %reset : i1, in %8 : i1) {
+    //CHECK:  %1 = seq.from_clock %clock
+    %0 = seq.from_clock %clock 
+    //CHECK:  %true = hw.constant true
+    %true = hw.constant true
+
+    //CHECK:  %false = hw.constant false
+    //CHECK:  %true_0 = hw.constant true
+    //CHECK:  %2 = comb.or %reset, %hbr : i1
+    //CHECK:  %hbr = seq.compreg sym @hbr  %2, %clock powerOn %false : i1  
+    %9 = verif.has_been_reset %0, sync %reset
+
+    //CHECK:  %3 = comb.xor %reset, %true_0 : i1
+    //CHECK:  %4 = comb.and %hbr, %3 : i1
+    //CHECK:  %5 = comb.xor bin %4, %true : i1
+    %10 = comb.xor bin %9, %true : i1
+
+    //CHECK:  %6 = hw.wire %0 : i1
```

Comment:
Not sure, but it's getting generated somewhere outside of my pipeline

Reply:
I would delete the `hw.wire` lines from the test, since your conversion doesn't explicitly do anything with them, so they don't add anything to the test.